plaso/parsers/mac_keychain.py

```diff
@@ -183,298 +142,719 @@ class KeychainParser(interface.FileObjectParser):
       'imap': 'imap',
       'http': 'http'}
 
-  def _ReadEntryApplication(self, parser_mediator, file_object):
-    """Extracts the information from an application password entry.
+  # TODO
```

Comment:
Todo what? ;)

Reply:
this was/is my refactor marker, I've removed it.